tutorials/examples/basic_image_array.py

```diff
@@ -0,0 +1,174 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import misc
+from skimage.measure import block_reduce
```

Comment:
Hehe…

Reply:
lol, well, that's embarrassing! I used ASPIRE to downsample instead. (This tutorial takes too long at full res).